### DISCUSSION TITLE
Running Laravel

### DISCUSSION THREAD

**Discussion by goparklubanic** [tomsik68/docker-xampp#64] [Q&A]
I need guidance to run laravel project in this container. I have put the project in /www folder, I need to be able call the project use some domain name http://laravel.local

**Comment by tomsik68** [comment]
Hey, thank you for being interested in this project! If you just want to run your Laravel project as simple as possible, can you please try [Laravel Sail](https://laravel.com/docs/11.x/installation#docker-installation-using-sail) first? It's a more complete solution which focuses on running Laravel projects locally in Docker. Apart from that, it offers comprehensive documentation which makes it much better suited for your use case.
